extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/OtlpExporterConfig.java

```diff
@@ -27,6 +28,17 @@
         @ConfigItem()
         public Optional<String> endpoint;
 
+        /**
+         * Key-value pairs to be used as headers associated with gRPC requests.
```

Comment:
The comment needs updating now that it's no longer a comma-separated list in a single string

Reply:
Please review the updated documentation, I saw benefits in mentioning the `OTEL_EXPORTER_OTLP_HEADERS` for the discoverability of this configuration parameter as `OTEL_EXPORTER_OTLP_HEADERS` is the standard environment variables practitioners are familiar with when instrumenting application with OpenTelemetry agents.
Please tell me if this is helpful or creating more confusion?

Reply:
I think it's ok